Formula/minicom.rb

```diff
@@ -20,15 +21,26 @@ class Minicom < Formula
     sha256 x86_64_linux:   "7d1b0aae1f169968d42e4dea644dff5a4f18010b59b334439aa2bd276c6e913a"
   end
 
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "gettext"
```

Comment:
Put these inside a `head` block, since we don't want them to be dependencies when not installing `HEAD`.

Reply:
I believe all of these are dependencies even when building the stable 2.8 sources

Reply:
That can't be true, because we were able to build 2.8 without these dependencies.

Reply:
My bad, I just noticed the build instructions are different for a release vs development version of `minicom` , I'll update that